editor/editor_sectioned_inspector.cpp

```diff
@@ -308,7 +311,7 @@ EditorInspector *SectionedInspector::get_inspector() {
 }
 
 SectionedInspector::SectionedInspector() :
-		obj(-1),
+		obj(UINT64_MAX),
```

Comment:
No... That's obviously not what the code expects.

Reply:
`core/object.cpp` seems to use 0 as the invalid valid for ObjectID, so this should be used here too (and in the other code assigning `-1` to `obj` when not editing an object.